wiremock/src/main/java/PatientDemographicsServiceClient.java

```diff
@@ -37,7 +38,7 @@ public Map<String, Object> patient(String nhsNumber) throws Exception {
 
         String patientData = getPatientData(nhsNumber, accessToken);
 
-        return new ObjectMapper().readValue(patientData, HashMap.class);
+        return new ObjectMapper().readValue(patientData, new TypeReference<>() {});
```

Comment:
The part which is new TypeReference<>() {} is actually new TypeReference<Map<String, Object>>() {}  but you can omit the Map<String, Object>> if it can be inferred.

This is a way to provide full type information to Jackson when deserializing, avoiding unchecked warnings. It's a generic safe way to tell the object mapper what it is being deserialized into